src/status_im2/contexts/wallet/add_address_to_watch/component_spec.cljs

```diff
@@ -13,11 +13,12 @@
 
 (h/describe "select address for watch only account"
   (h/test "validation messages show for already used addressed"
-    (setup-subs {:wallet/scanned-address      nil
-                 :wallet/addresses            (set
-                                               ["0x12E838Ae1f769147b12956485dc56e57138f3AC8"
-                                                "0x22E838Ae1f769147b12956485dc56e57138f3AC8"])
-                 :profile/customization-color :blue})
+    (setup-subs {:wallet/scanned-address              nil
+                 :wallet/addresses                    (set
+                                                       ["0x12E838Ae1f769147b12956485dc56e57138f3AC8"
+                                                        "0x22E838Ae1f769147b12956485dc56e57138f3AC8"])
```

Comment:
We could just hardcode the set here, right?
I mean `#{"asd" "fgh"}`

I know you didn't add this, but just wondering

Reply:
Yes @ulisesmac, we can hardcode the set instead of the wrapper with the set.

I will update it 👍